packages/material-ui/src/Slider/Slider.js

```diff
@@ -720,7 +721,17 @@ Slider.propTypes = {
   /**
    * The label of the slider.
    */
-  'aria-label': PropTypes.string,
+  'aria-label': chainPropTypes(PropTypes.string, props => {
+    const range = Array.isArray(props.value || props.defaultValue);
+
+    if (range && props['aria-label']) {
```

Comment:
```suggestion
    if (range && props['aria-label'] != null) {
```

Don't want to overload string types as well (since `Boolean("") === false`)